packages/onboarding/src/hooks/use-persisted-state.ts

```diff
@@ -23,6 +23,20 @@ function getPersistedState( key: string, storage: Storage, TTL: number ) {
 	}
 }
 
+/**
+ * Clears all persisted state created by useStepPersistedState
+ * @param storage The storage to clear (defaults to localStorage)
+ */
+function clearStepPersistedState( storage: Storage = localStorage ): void {
```

Comment:
The API would feel a lot cleaner to me if this function was just exported directly. It doesn't depend on any React state, so it doesn't feel like it needs to be returned by a hook.

The fact that the only time it's actually used, the first two things returned from the hook aren't used, it kinda shows how it doesn't need to be colocated with the `useStepPersistedState` hook for it to be useful.

Reply:
I thought about this while working with https://github.com/Automattic/wp-calypso/pull/97920 and agree that this doesn't need to live inside the `useStepPersistedState` hook.

I really like how the two functions are now in the same file, close together.

Another reason is that if it's inside the hook, we’d run the rest of the hook’s code unnecessarily even when all we need is `clearStepPersistedState`.

So +1 for exporting this function directly in this file ❤️